src/services/index.ts

```diff
@@ -207,6 +239,40 @@ export const getSavedFiltersAds = async (user: IUser): Promise<ISavedFilterAds[]
   }
 };
 
+export const calculateAllLiquidity = async () => {
+  const bodyTypes = await getAllBodyTypes();
+  const models = await getModels();
+  const totalSold = await AdService.countSoldAds();
+
+  for (const model of models) {
+    for (const bodyType of bodyTypes) {
+      const totalSoldInConfig = await AdService.countSoldWithFilter(model.id, bodyType.id);
+      const ads = await getAds({ modelId: [model.id], bodyTypeId: [bodyType.id] }, null);
+      const adPrices = ads.map((item: any) => item.price);
+      adPrices.sort((item: any, nextItem: any) => {
```

Comment:
Lodash has a method not to do this like that. You should somehow specify a path to a property. Or do the arrow function returning a value to sort by.